src/components/projects/create-wp-form/create-wp-form.tsx

```diff
@@ -5,8 +5,28 @@
 
 import { useState } from "react";
 import { useHistory } from "react-router-dom";
+import { isProject, validateWBS, WbsNumber } from "utils";
 import CreateWPFormView from "./create-wp-form/create-wp-form";
 
+const validate = (e: any) => {
```

Comment:
I think we can simplify things and put this logic into the handle submit function. I also think it's totally fine to fold it together and not have a custom error message. Check the implementations but I recall the validate one being written to throw if not valid.